client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -106,7 +106,8 @@ const BrewItem = createClass({
 			<div className='info'>
 				{brew.tags ? <>
 					<span className='brewTags' title={`Tags:\n${brew.tags.join('\n')}`}>
-						<i className='fas fa-tags'/> {brew.tags.join(', ')}
+						<i className='fas fa-tags'/>
+						{brew.tags.map((tag, idx)=>{return <><span>{tag}</span>{'\n'}</>;})}
```

Comment:
Drop the `{'\n'}` because that introduces white space between the spans in the browser rendered output which messes with attempts to control spacing the spans with `margin-left: 2px;`.

Reply:
whoops, forgot to push the last change. Already had this but good eye. Last commit is pushed up now.